--- wireshark-3.0.0/CMakeLists.txt~	2019-02-28 18:56:38.000000000 +0100
+++ wireshark-3.0.0/CMakeLists.txt	2019-03-28 22:59:33.037351621 +0100
@@ -1473,7 +1473,7 @@
 
 set( prefix "${CMAKE_INSTALL_PREFIX}" )
 set( exec_prefix "\${prefix}" )
-set( libdir "\${exec_prefix}/${CMAKE_INSTALL_LIBDIR}" )
+set( libdir "${CMAKE_INSTALL_LIBDIR}" )
 set( includedir  "\${prefix}/include" )
 set( plugindir "\${libdir}/wireshark/${PLUGIN_VERSION_DIR}" )
 
