--- plugins.c.orig	Tue Feb  1 05:56:34 2000
+++ plugins.c	Wed Feb  9 15:01:49 2000
@@ -61,7 +61,7 @@
 /* linked list of all plugins */
 plugin *plugin_list;
 
-static gchar std_plug_dir[] = "/usr/lib/ethereal/plugins/0.8";
+static gchar std_plug_dir[] = "/usr/X11R6/lib/ethereal/plugins/0.8";
 static gchar local_plug_dir[] = "/usr/local/lib/ethereal/plugins/0.8";
 static gchar *user_plug_dir = NULL;
 static gchar *plugin_status_file = NULL;
