--- wireshark-1.0.0/epan/Makefile.am.orig	2008-04-01 05:57:15.796355855 +0200
+++ wireshark-1.0.0/epan/Makefile.am	2008-04-01 05:59:06.288761964 +0200
@@ -129,15 +129,17 @@
 	libwireshark_asmopt.la crypt/libairpdcap.la ftypes/libftypes.la \
 	dfilter/libdfilter.la dissectors/libcleandissectors.la \
 	dissectors/libdissectors.la dissectors/libasndissectors.la \
-	dissectors/libpidldissectors.la $(wslua_lib) @SOCKET_LIBS@ @NSL_LIBS@ \
+	dissectors/libpidldissectors.la ${top_builddir}/wiretap/libwiretap.la \
+	$(wslua_lib) @SOCKET_LIBS@ @NSL_LIBS@ \
 	@ADNS_LIBS@ @LIBGCRYPT_LIBS@ @LIBGNUTLS_LIBS@ @LIBICONV@ @KRB5_LIBS@ \
-	@SSL_LIBS@ @LIBSMI_LDFLAGS@ -lm
+	@SSL_LIBS@ @LIBSMI_LDFLAGS@ -lm -lgmodule-2.0
 libwireshark_la_DEPENDENCIES = @G_ASCII_STRCASECMP_LO@ @G_ASCII_STRTOULL_LO@ \
	@INET_ATON_LO@ @INET_PTON_LO@ @INET_NTOP_LO@ libwireshark_generated.la \
 	libwireshark_asmopt.la crypt/libairpdcap.la ftypes/libftypes.la \
 	dfilter/libdfilter.la dissectors/libcleandissectors.la \
 	dissectors/libdissectors.la dissectors/libasndissectors.la \
-	dissectors/libpidldissectors.la $(wslua_lib)
+	dissectors/libpidldissectors.la ${top_builddir}/wiretap/libwiretap.la \
+	$(wslua_lib)
 
 #EXTRA_PROGRAMS = reassemble_test
 #reassemble_test_LDADD = $(GLIB_LIBS)
--- wireshark-1.0.0/wiretap/Makefile.am.orig	2008-04-01 05:56:08.300994660 +0200
+++ wireshark-1.0.0/wiretap/Makefile.am	2008-04-01 05:56:47.298314463 +0200
@@ -67,7 +67,7 @@
 	$(GENERATOR_FILES) 	\
 	$(GENERATED_FILES)
 
-libwiretap_la_LIBADD = libwiretap_generated.la
+libwiretap_la_LIBADD = libwiretap_generated.la -lglib-2.0
 libwiretap_la_DEPENDENCIES = libwiretap_generated.la
 
 RUNLEX = $(top_srcdir)/tools/runlex.sh
