--- wireshark-1.2.0/epan/Makefile.am~	2009-06-16 06:03:39.000000000 +0200
+++ wireshark-1.2.0/epan/Makefile.am	2009-06-23 01:19:15.746887934 +0200
@@ -127,16 +127,17 @@
 	@INET_ATON_LO@ @INET_PTON_LO@ @INET_NTOP_LO@ libwireshark_generated.la \
 	libwireshark_asmopt.la crc/libcrc.la crypt/libairpdcap.la \
 	ftypes/libftypes.la dfilter/libdfilter.la dissectors/libdissectors.la \
-	dissectors/libdirtydissectors.la $(wslua_lib) @SOCKET_LIBS@ @NSL_LIBS@ \
+	dissectors/libdirtydissectors.la ${top_builddir}/wiretap/libwiretap.la \
+	$(wslua_lib) @SOCKET_LIBS@ @NSL_LIBS@ \
 	@C_ARES_LIBS@ @ADNS_LIBS@ @LIBGCRYPT_LIBS@ @LIBGNUTLS_LIBS@ \
 	@KRB5_LIBS@ @SSL_LIBS@ @LIBSMI_LDFLAGS@ @GEOIP_LIBS@ \
-	${top_builddir}/wsutil/libwsutil.la -lm
+	${top_builddir}/wsutil/libwsutil.la -lm -lgmodule-2.0
 
 libwireshark_la_DEPENDENCIES = \
 	@INET_ATON_LO@ @INET_PTON_LO@ @INET_NTOP_LO@ libwireshark_generated.la \
 	libwireshark_asmopt.la crc/libcrc.la crypt/libairpdcap.la \
 	ftypes/libftypes.la dfilter/libdfilter.la dissectors/libdissectors.la \
-	dissectors/libdirtydissectors.la $(wslua_lib) \
+	dissectors/libdirtydissectors.la $(wslua_lib) ${top_builddir}/wiretap/libwiretap.la \
 	${top_builddir}/wsutil/libwsutil.la
 
 #EXTRA_PROGRAMS = reassemble_test
