--- wireshark-1.6.7/wireshark.desktop	2012-06-02 18:29:44.784985048 +0300
+++ wireshark-1.6.7/wireshark.desktop	2012-06-02 18:30:27.236953001 +0300
@@ -67,6 +67,8 @@
 Comment=Network traffic analyzer
 Comment[fi]=Verkkoliikenne analysaattori
 Comment[fr]=Analyseur de trafic réseau
+Comment[ja]=The Wireshark ネットワーク・アナライザー
+Comment[pl]=Wireshark - analizator ruchu w sieci
 Comment[sv]=Nätverkstrafikanalysator
 Icon=wireshark.png
 TryExec=wireshark
@@ -78 +80 @@
-Categories=System;Monitor;GTK;
+Categories=GTK;Network;
