diff -ruN wireshark-2.0.1.orig/acinclude.m4 wireshark-2.0.1/acinclude.m4
--- wireshark-2.0.1.orig/acinclude.m4	2015-12-29 22:49:18.000000000 +0100
+++ wireshark-2.0.1/acinclude.m4	2016-01-11 00:38:48.501985907 +0100
@@ -389,7 +389,7 @@
 	    # work.
 	    #
 	    AC_MSG_CHECKING(for broken pcap-config)
-	    case "`\"$PCAP_CONFIG\" --libs`" in
+	    case "`$PCAP_CONFIG --libs`" in
 
 	    "-L "*)
 	      #
@@ -413,6 +413,7 @@
 	    # for libpcap.
 	    #
 	    CPPFLAGS="$CPPFLAGS `\"$PCAP_CONFIG\" --cflags`"
+	    CPPFLAGS="$CPPFLAGS `$PCAP_CONFIG --cflags`"
 	  else
 	    #
 	    # Didn't find it; we have to look for libpcap ourselves.
@@ -481,9 +482,9 @@
 	  # libraries libpcap needs.
 	  #
 	  if test x$enable_static = xyes; then
-	    PCAP_LIBS="`\"$PCAP_CONFIG\" --libs --static`"
+	    PCAP_LIBS="`$PCAP_CONFIG --libs --static`"
 	  else
-	    PCAP_LIBS="`\"$PCAP_CONFIG\" --libs`"
+	    PCAP_LIBS="`$PCAP_CONFIG --libs`"
 	  fi
 	  AC_DEFINE(HAVE_LIBPCAP, 1, [Define to use libpcap library])
 	else
