--- wireshark-1.2.4/wireshark-kde.desktop	2009-12-09 10:09:05.765409184 +0200
+++ wireshark-1.2.4/wireshark-kde.desktop	2009-12-09 10:18:48.798894254 +0200
@@ -71,7 +71,11 @@
+Comment[ja]=The Wireshark ネットワーク・アナライザー
+Comment[pl]=Wireshark - analizator ruchu w sieci
 Icon=wireshark.png
 TryExec=wireshark
 Exec=wireshark
 Terminal=false
+X-KDE-SubstituteUID=true
+OnlyShowIn=KDE;
 # Categorie entry according to version 1.0 of
 # http://www.freedesktop.org/wiki/Specifications/menu-spec
-Categories=System;Monitor;GTK;
+Categories=GTK;Network;
--- wireshark-1.2.4/wireshark.desktop~	2009-11-16 22:33:26.000000000 +0200
+++ wireshark-1.2.4/wireshark.desktop	2009-12-09 10:16:55.333281083 +0200
@@ -68,10 +68,13 @@
 Comment[fi]=Verkkoliikenne analysaattori
 Comment[fr]=Analyseur de trafic réseau
 Comment[sv]=Nätverkstrafikanalysator
+Comment[ja]=The Wireshark ネットワーク・アナライザー
+Comment[pl]=Wireshark - analizator ruchu w sieci
 Icon=wireshark.png
 TryExec=wireshark
-Exec=wireshark
+Exec=gksu -u root wireshark
+NotShowIn=KDE;
 Terminal=false
 # Categorie entry according to version 1.0 of
 # http://www.freedesktop.org/wiki/Specifications/menu-spec
-Categories=System;Monitor;GTK;
+Categories=GTK;Network;
