--- wireshark-4.4.1/extcap/falcodump.cpp.orig	2024-10-16 21:35:13.731822931 +0200
+++ wireshark-4.4.1/extcap/falcodump.cpp	2024-10-16 21:36:20.628127189 +0200
@@ -1179,7 +1179,7 @@ int main(int argc, char **argv)
                     ws_warning("%s", init_err.c_str());
                     goto end;
                 }
-                inspector.open_plugin(extcap_conf->interface, plugin_source);
+                inspector.open_plugin(extcap_conf->interface, plugin_source, sinsp_plugin_platform::SINSP_PLATFORM_GENERIC);
                 // scap_dump_open handles "-"
             } catch (sinsp_exception &e) {
                 ws_warning("%s", e.what());
--- wireshark-4.4.1/plugins/epan/falco_bridge/sinsp-span.cpp.orig	2024-10-16 21:35:13.831253554 +0200
+++ wireshark-4.4.1/plugins/epan/falco_bridge/sinsp-span.cpp	2024-10-16 22:11:50.924659565 +0200
@@ -443,13 +443,13 @@ void create_sinsp_syscall_source(sinsp_s
                 if (!sfc) {
                     continue;
                 }
-                if (strcmp(ffi->m_name, "evt.category") == 0) {
+                if (ffi->m_name == "evt.category") {
                     ssi->evt_category_idx = ssi->syscall_filter_fields.size();
                 }
-                if (strcmp(ffi->m_name, "evt.cpu") == 0) {
+		else if (ffi->m_name == "evt.cpu") {
                     ssi->cpu_id_idx = (uint16_t) ssi->syscall_filter_fields.size();
                 }
-                if (strcmp(ffi->m_name, "proc.pid") == 0) {
+		else if (ffi->m_name == "proc.pid") {
                     ssi->proc_id_idx = (uint16_t) ssi->syscall_filter_fields.size();
                 }
                 sfc->parse_field_name(ffi->m_name, true, false);
@@ -559,18 +559,18 @@ bool get_sinsp_source_field_info(sinsp_s
 
     if (ssi->source) {
         ffi = &ssi->source->fields()[field_num];
-        g_strlcpy(field->abbrev, ffi->m_name, sizeof(field->abbrev));
+        g_strlcpy(field->abbrev, ffi->m_name.c_str(), sizeof(field->abbrev));
     } else {
         ffi = ssi->syscall_filter_fields[field_num];
         if (ssi->field_to_category[field_num] == SSC_OTHER) {
-            snprintf(field->abbrev, sizeof(field->abbrev), FALCO_FIELD_NAME_PREFIX "%s", ffi->m_name);
+            snprintf(field->abbrev, sizeof(field->abbrev), FALCO_FIELD_NAME_PREFIX "%s", ffi->m_name.c_str());
         } else {
-            g_strlcpy(field->abbrev, ffi->m_name, sizeof(field->abbrev));
+            g_strlcpy(field->abbrev, ffi->m_name.c_str(), sizeof(field->abbrev));
         }
     }
 
-    g_strlcpy(field->display, ffi->m_display, sizeof(field->display));
-    g_strlcpy(field->description, ffi->m_description, sizeof(field->description));
+    g_strlcpy(field->display, ffi->m_display.c_str(), sizeof(field->display));
+    g_strlcpy(field->description, ffi->m_description.c_str(), sizeof(field->description));
 
     field->is_hidden = ffi->m_flags & EPF_TABLE_ONLY;
     field->is_conversation = ffi->m_flags & EPF_CONVERSATION;
