diff -ruNp wireshark-2.2.0.orig/epan/dissectors/asn1/kerberos/packet-kerberos-template.c wireshark-2.2.0/epan/dissectors/asn1/kerberos/packet-kerberos-template.c
--- wireshark-2.2.0.orig/epan/dissectors/asn1/kerberos/packet-kerberos-template.c	2016-09-07 18:59:02.000000000 +0200
+++ wireshark-2.2.0/epan/dissectors/asn1/kerberos/packet-kerberos-template.c	2016-09-12 15:31:55.598702256 +0200
@@ -810,10 +810,6 @@ decrypt_krb5_data(proto_tree *tree, pack
 #define KRB5_MSG_ERROR			30	/* KRB-ERROR type */
 
 /* encryption type constants */
-#define KRB5_ENCTYPE_NULL		0
-#define KRB5_ENCTYPE_DES_CBC_CRC	1
-#define KRB5_ENCTYPE_DES_CBC_MD4	2
-#define KRB5_ENCTYPE_DES_CBC_MD5	3
 #define KRB5_ENCTYPE_DES_CBC_RAW	4
 #define KRB5_ENCTYPE_DES3_CBC_SHA	5
 #define KRB5_ENCTYPE_DES3_CBC_RAW	6
@@ -825,9 +821,6 @@ decrypt_krb5_data(proto_tree *tree, pack
 #define KRB5_ENCTYPE_RSA_ENV		13
 #define KRB5_ENCTYPE_RSA_ES_OEAP_ENV	14
 #define KRB5_ENCTYPE_DES_EDE3_CBC_ENV	15
-#define KRB5_ENCTYPE_DES3_CBC_SHA1	16
-#define KRB5_ENCTYPE_AES128_CTS_HMAC_SHA1_96 17
-#define KRB5_ENCTYPE_AES256_CTS_HMAC_SHA1_96 18
 #define KRB5_ENCTYPE_DES_CBC_MD5_NT	20
 #define KERB_ENCTYPE_RC4_HMAC		23
 #define KERB_ENCTYPE_RC4_HMAC_EXP	24
diff -ruNp wireshark-2.2.0.orig/epan/dissectors/asn1/kerberos/packet-kerberos-template.h wireshark-2.2.0/epan/dissectors/asn1/kerberos/packet-kerberos-template.h
--- wireshark-2.2.0.orig/epan/dissectors/asn1/kerberos/packet-kerberos-template.h	2016-09-07 18:59:02.000000000 +0200
+++ wireshark-2.2.0/epan/dissectors/asn1/kerberos/packet-kerberos-template.h	2016-09-12 15:31:55.171703765 +0200
@@ -101,10 +101,6 @@ void read_keytab_file_from_preferences(v
 #endif /* HAVE_KERBEROS */
 
 /* encryption type constants */
-#define KRB5_ENCTYPE_NULL		0
-#define KRB5_ENCTYPE_DES_CBC_CRC	1
-#define KRB5_ENCTYPE_DES_CBC_MD4	2
-#define KRB5_ENCTYPE_DES_CBC_MD5	3
 #define KRB5_ENCTYPE_DES_CBC_RAW	4
 #define KRB5_ENCTYPE_DES3_CBC_SHA	5
 #define KRB5_ENCTYPE_DES3_CBC_RAW	6
@@ -116,9 +112,6 @@ void read_keytab_file_from_preferences(v
 #define KRB5_ENCTYPE_RSA_ENV		13
 #define KRB5_ENCTYPE_RSA_ES_OEAP_ENV	14
 #define KRB5_ENCTYPE_DES_EDE3_CBC_ENV	15
-#define KRB5_ENCTYPE_DES3_CBC_SHA1	16
-#define KRB5_ENCTYPE_AES128_CTS_HMAC_SHA1_96 17
-#define KRB5_ENCTYPE_AES256_CTS_HMAC_SHA1_96 18
 #define KRB5_ENCTYPE_DES_CBC_MD5_NT	20
 #define KERB_ENCTYPE_RC4_HMAC		23
 #define KERB_ENCTYPE_RC4_HMAC_EXP	24
diff -ruNp wireshark-2.2.0.orig/epan/dissectors/packet-kerberos.c wireshark-2.2.0/epan/dissectors/packet-kerberos.c
--- wireshark-2.2.0.orig/epan/dissectors/packet-kerberos.c	2016-09-07 18:59:03.000000000 +0200
+++ wireshark-2.2.0/epan/dissectors/packet-kerberos.c	2016-09-12 15:31:56.421699346 +0200
@@ -1069,10 +1069,6 @@ decrypt_krb5_data(proto_tree *tree, pack
 #define KRB5_MSG_ERROR			30	/* KRB-ERROR type */
 
 /* encryption type constants */
-#define KRB5_ENCTYPE_NULL		0
-#define KRB5_ENCTYPE_DES_CBC_CRC	1
-#define KRB5_ENCTYPE_DES_CBC_MD4	2
-#define KRB5_ENCTYPE_DES_CBC_MD5	3
 #define KRB5_ENCTYPE_DES_CBC_RAW	4
 #define KRB5_ENCTYPE_DES3_CBC_SHA	5
 #define KRB5_ENCTYPE_DES3_CBC_RAW	6
@@ -1084,9 +1080,6 @@ decrypt_krb5_data(proto_tree *tree, pack
 #define KRB5_ENCTYPE_RSA_ENV		13
 #define KRB5_ENCTYPE_RSA_ES_OEAP_ENV	14
 #define KRB5_ENCTYPE_DES_EDE3_CBC_ENV	15
-#define KRB5_ENCTYPE_DES3_CBC_SHA1	16
-#define KRB5_ENCTYPE_AES128_CTS_HMAC_SHA1_96 17
-#define KRB5_ENCTYPE_AES256_CTS_HMAC_SHA1_96 18
 #define KRB5_ENCTYPE_DES_CBC_MD5_NT	20
 #define KERB_ENCTYPE_RC4_HMAC		23
 #define KERB_ENCTYPE_RC4_HMAC_EXP	24
diff -ruNp wireshark-2.2.0.orig/epan/dissectors/packet-kerberos.h wireshark-2.2.0/epan/dissectors/packet-kerberos.h
--- wireshark-2.2.0.orig/epan/dissectors/packet-kerberos.h	2016-09-07 18:59:03.000000000 +0200
+++ wireshark-2.2.0/epan/dissectors/packet-kerberos.h	2016-09-12 15:31:56.003700824 +0200
@@ -109,10 +109,6 @@ void read_keytab_file_from_preferences(v
 #endif /* HAVE_KERBEROS */
 
 /* encryption type constants */
-#define KRB5_ENCTYPE_NULL		0
-#define KRB5_ENCTYPE_DES_CBC_CRC	1
-#define KRB5_ENCTYPE_DES_CBC_MD4	2
-#define KRB5_ENCTYPE_DES_CBC_MD5	3
 #define KRB5_ENCTYPE_DES_CBC_RAW	4
 #define KRB5_ENCTYPE_DES3_CBC_SHA	5
 #define KRB5_ENCTYPE_DES3_CBC_RAW	6
@@ -124,9 +120,6 @@ void read_keytab_file_from_preferences(v
 #define KRB5_ENCTYPE_RSA_ENV		13
 #define KRB5_ENCTYPE_RSA_ES_OEAP_ENV	14
 #define KRB5_ENCTYPE_DES_EDE3_CBC_ENV	15
-#define KRB5_ENCTYPE_DES3_CBC_SHA1	16
-#define KRB5_ENCTYPE_AES128_CTS_HMAC_SHA1_96 17
-#define KRB5_ENCTYPE_AES256_CTS_HMAC_SHA1_96 18
 #define KRB5_ENCTYPE_DES_CBC_MD5_NT	20
 #define KERB_ENCTYPE_RC4_HMAC		23
 #define KERB_ENCTYPE_RC4_HMAC_EXP	24
