--- wireshark-0.99.6/acinclude.m4~	2007-09-04 14:52:13.000000000 +0200
+++ wireshark-0.99.6/acinclude.m4	2007-09-04 14:56:04.000000000 +0200
@@ -486,7 +486,7 @@
 	ac_CFLAGS_saved="$CFLAGS"
 	AC_WIRESHARK_PCAP_BREAKLOOP_TRY_LINK
 	if test "x$ws_breakloop_compiled" = "xyes"; then
-	  CFLAGS="$CFLAGS -Werror -Wimplicit"
+	  CFLAGS="$CFLAGS -Wimplicit"
 	  AC_WIRESHARK_PCAP_BREAKLOOP_TRY_LINK
 	  if test "x$ws_breakloop_compiled" = "xyes"; then
 	    AC_MSG_RESULT(yes)
