--- wireshark-1.2.7/acinclude.m4~	2010-04-01 14:02:02.000000000 +0300
+++ wireshark-1.2.7/acinclude.m4	2010-04-01 14:03:16.846003020 +0300
@@ -304,7 +304,7 @@
 	  # work.
 	  #
 	  AC_MSG_CHECKING(for broken pcap-config)
-	    case "`\"$PCAP_CONFIG\" --libs`" in
+	    case "`$PCAP_CONFIG --libs`" in
 
 	    "-L "*)
 	      #
@@ -397,9 +397,9 @@
 	  # libraries libpcap needs.
 	  #
 	  if test x$enable_static = xyes; then
-	    PCAP_LIBS="`\"$PCAP_CONFIG\" --libs --static`"
+	    PCAP_LIBS="`$PCAP_CONFIG --libs --static`"
 	  else
-	    PCAP_LIBS="`\"$PCAP_CONFIG\" --libs`"
+	    PCAP_LIBS="`$PCAP_CONFIG --libs`"
 	  fi
 	  AC_DEFINE(HAVE_LIBPCAP, 1, [Define to use libpcap library])
 	else
